staging/src/k8s.io/kubectl/pkg/cmd/taint/taint.go

```diff
@@ -241,20 +241,29 @@ func (o TaintOptions) Validate() error {
 	}
 
 	// check the format of taint args and checks removed taints aren't in the new taints list
-	var conflictTaints []string
+	var conflictTaints []v1.Taint
 	for _, taintAdd := range o.taintsToAdd {
 		for _, taintRemove := range o.taintsToRemove {
 			if taintAdd.Key != taintRemove.Key {
 				continue
 			}
 			if len(taintRemove.Effect) == 0 || taintAdd.Effect == taintRemove.Effect {
-				conflictTaint := fmt.Sprintf("{\"%s\":\"%s\"}", taintRemove.Key, taintRemove.Effect)
```

Comment:
Let's scrap the json altogether here and just make these `key=value`.

Reply:
We don't even output the conflicts for labels 😅

https://github.com/kubernetes/kubernetes/blob/fc85deff7faef7f349742bba599696175ba920d4/staging/src/k8s.io/kubectl/pkg/cmd/label/label.go#L431

Reply:
Thank you for reviewing the PR. I have made the requested changes.

Reply:
@eddiezane before issuing the lgtm here, I think that your point in the second review is: we should also print the "conflictTaints" right?

@manugupt1 maybe doing something like:

return nil, nil, fmt.Errorf("can not both modify and remove a label in the same command: %v", conflictTaint) 

or creating a pretifier just for this array would be really helpful for users :D

Reply:
@rikatz sorry for the confusion. The return error is happening already below

https://github.com/kubernetes/kubernetes/blob/5f35d48104cf285511adffeed8bee79207ee00cc/staging/src/k8s.io/kubectl/pkg/cmd/taint/taint.go#L257

Reply:
Thanks Eddie, I didn't went to check above :)